### ISSUE TITLE
Keyboard is not showing when focus on text field 

### ISSUE THREAD

**Issue by maliknaqibullah** [Breens-Mbaka/Searchable-Dropdown-Menu-Jetpack-Compose#38]
after following all the given steps in the example provided I faced this issue which does not pop up the keyboard when focus on search text field . Is anyone else face this issue too?

**Comment by Breens-Mbaka** [comment]
Hi @maliknaqibullah you must click on the search text field to open the keyboard. See demo below
![untitled](https://github.com/Breens-Mbaka/Searchable-Dropdown-Menu-Jetpack-Compose/assets/72180010/7d05c333-6486-4043-926e-cde3f945fcfb)

**Comment by Breens-Mbaka** [comment]
Hi @maliknaqibullah after clicking on the dropdown menu for you to open the keyboard to search, you have to click on the search text field. See the video demo ➡️ [demo.webm](https://github.com/Breens-Mbaka/Searchable-Dropdown-Menu-Jetpack-Compose/assets/72180010/0beeeb04-02bc-48ec-84c3-01ecb6cfe777)

**Comment by maliknaqibullah** [comment]
@Breens-Mbaka thank you for replying.  Yeah I am trying that but does not work. The only think I changed is applicationContext which didn't work

**Comment by maliknaqibullah** [comment]
(https://github.com/Breens-Mbaka/Searchable-Dropdown-Menu-Jetpack-Compose/assets/33382700/1b5a367f-0c20-43f8-b263-c83394cf849a)

**Comment by Breens-Mbaka** [comment]
What Android version are you using and what device are you running on?

**Comment by maliknaqibullah** [comment]
Android version 13
Device Samsung A33 5G

**Comment by Breens-Mbaka** [comment]
Try clicking inside the search text field
<img width="381" alt="Screenshot 2023-06-07 at 18 21 36" src="https://github.com/Breens-Mbaka/Searchable-Dropdown-Menu-Jetpack-Compose/assets/72180010/f73ce303-0d85-4902-9c1c-32555ad890c3">

**Comment by maliknaqibullah** [comment]
yeah I am touching right there in the search text field and it does not show up the keyboard.

**Comment by Breens-Mbaka** [comment]
> yeah I am touching right there in the search text field and it does not show up on the keyboard.

That's weird you can try to use the latest version of the app, meanwhile I'll have a look at the issue, or if you want you can create a PR for it if you want

**Comment by ashishkr2534** [comment]
@Breens-Mbaka I too have the same isssue . i create a whole new searchable dropdown menu by myself but the keyboard did not poped up. i tried many ways but couldn't make it work.

At last i came across your repo, hoping it would work seeing nice review but sadly it didn't , using andorid 12. 

If possible - do give a feed on this, much appreciated.

**Comment by ashishkr2534** [comment]
I checked on this error. dont exactly got how. but it arises on devices running above android 10.

**Comment by Breens-Mbaka** [comment]
Okay thanks for this, let me investigate and find whats the issue on this

**Comment by Breens-Mbaka** [comment]
> @Breens-Mbaka I too have the same isssue . i create a whole new searchable dropdown menu by myself but the keyboard did not poped up. i tried many ways but couldn't make it work.
> 
> At last i came across your repo, hoping it would work seeing nice review but sadly it didn't , using andorid 12.
> 
> If possible - do give a feed on this, much appreciated.

Hi, @ashishkr2534 would you mind telling me what version of the SearchableDropDownMenu library you're using currently in your project that has this issue?

**Comment by Breens-Mbaka** [comment]
> yeah I am touching right there in the search text field and it does not show up the keyboard.



> after following all the given steps in the example provided I faced this issue which does not pop up the keyboard when focus on search text field . Is anyone else face this issue too?

Hi, @maliknaqibullah  would you mind telling me what version of the SearchableDropDownMenu library you're using currently in your project that has this issue?

**Comment by ashishkr2534** [comment]
@Breens-Mbaka  version 0.2.9

**Comment by Breens-Mbaka** [comment]
Hi @ashishkr2534 @maliknaqibullah, I just replicated the issue I wasn't able to open the keyboard in Android 13 and 12, can you confirm if this is the exact issue you're facing with this [video recording](https://github.com/Breens-Mbaka/Searchable-Dropdown-Menu-Jetpack-Compose/assets/72180010/3507e7e0-cd4e-4877-bf91-4ee1f711d60e)
